WalletWasabi.Fluent/ViewModels/CoinControl/Core/PocketCoinControlItemViewModel.cs

```diff
@@ -0,0 +1,23 @@
+using System.Linq;
+using WalletWasabi.Fluent.Models;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinControl.Core;
+
+internal class PocketCoinControlItemViewModel : CoinControlItemViewModelBase
+{
+	public PocketCoinControlItemViewModel(Pocket pocket)
+	{
+		var confirmationCount = pocket.Coins.Count();
+		var unconfirmedCount = pocket.Coins.Count(x => !x.Confirmed);
+		var allConfirmed = confirmationCount == unconfirmedCount;
+		IsBanned = pocket.Coins.Any(x => x.IsBanned);
+		ConfirmationStatus = allConfirmed ? "All coins are confirmed" : $"{unconfirmedCount} coins are waiting for confirmation";
```

Comment:
Swap the positions as it makes more sense.

Reply:
Switched.
L:13 => L:14
L:14 => L:13